src/components/Frame/Frame.tsx

```diff
@@ -43,6 +43,10 @@ export interface FrameProps {
    * @default false
    */
   showMobileNavigation?: boolean;
+  /** ID of the element you wish to skip to when clicking skip to content link */
+  skipToContentTargetId?: string;
```

Comment:
optionally this can just be removed and replaced with `this.skipToMainContentTargetNode.current.id` and get the id out of the `RefObject`. My main concern what the id might be `null` if someone doesn't have it so I went with the explicit route. However this 2 props might make it more confusing to use

Reply:
you know what i think i'm going to just switch to this approach i think the simpler api is better